wikimedia/wikipedia-ios
# Talk pages -  add formatting toolbar

**Phabricator:** https://phabricator.wikimedia.org/T315606

### Notes
* This PR adds a simple formatting toolbar to the talk page reply compose view and to the topic compose view controller
* The toolbar is a simple UIView containing the actions to add bold, italics, and link _wikitext_ formatting to the UITextView
* The add image action was not considered for this PR
* The link editing was also not considered

### Test Steps
1. On a talk page, tap reply to topic or create new topic
2. For bold and italics, you can select previously typed text or select the button before typing
3. For link insertion, it will display a new view, search for article and insert it
4. Post your topic and reply, and verify that the formatting works
5. To remove the formatting, select the typed word and click the same button - also works if nothing was typed
6. Make sure the find in page toolbar was not affected
